Postcss is not defined.

Hey there,

Recieving this error both [here]( https://npm.runkit.com/postcss-combine-duplicated-selectors) and on the CLI. Installed this package globally with the following command:

`npm install -g postcss-combine-duplicated-selectors`

Any ideas?

**Edit**:

After a closer look I realized that you probably need Postcss-cli for this (new to Node.js.) Now I recieve the following error:

`× Plugin Error: Cannot find module '*.css'` 
`× Plugin Error: Cannot find module 'style0.css'`

with the given command:

`postcss --use postcss-combine-duplicated-selectors *.css`

**Double Edit**: Switched the file name and the --use command around and it seems to work now. 

**Triple Edit**: Working solution: Install postcss-cli (which is probably common knowledge in the community, not sure.) Then swap around the use statement with the file name.

`postcss style.css --use postcss-combine-duplicated-selectors -o style1.css)`

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # Requirements
 
-In order to use this you will need to have [postcss](https://github.com/postcss/postcss) installed. Depending on whether or not you want to use the CLI you also need the [postcss-cli](https://github.com/postcss/postcss-cli) installed.
+In order to use this you will need to have [postcss](https://github.com/postcss/postcss) installed. Depending on whether or not you want to use the CLI you need to install [postcss-cli](https://github.com/postcss/postcss-cli).
 
 
 # Postcss combine duplicated selectors

--- a/README.md
+++ b/README.md
@@ -1,4 +1,9 @@
-# postcss combine duplicated selectors
+# Requirements
+
+In order to use this you will need to have [postcss](https://github.com/postcss/postcss) installed. Depending on whether or not you want to use the CLI you also need the [postcss-cli](https://github.com/postcss/postcss-cli) installed.
+
+
+# Postcss combine duplicated selectors
 
 <!-- current project status -->
 [![npm](https://img.shields.io/npm/v/postcss-combine-duplicated-selectors.svg)](https://www.npmjs.com/package/postcss-combine-duplicated-selectors)
@@ -36,9 +41,8 @@ postcss([require('postcss-combine-duplicated-selectors')])
 ```
 
 ### Using PostCSS CLI
-
 ``` sh
-postcss --use postcss-combine-duplicated-selectors *.css
+postcss style.css --use postcss-combine-duplicated-selectors -o newcss.css)
 ```
 
 ## Example

